build/media_source/plg_system_shortcut/js/shortcut.es6.js

```diff
@@ -0,0 +1,64 @@
+class Shortcut {
```

Comment:
Rename the class to `JoomlaShortcuts`

Reply:
No need for this. The file is an es module so everything here is encapsulated

Reply:
Ok cool  @dgrammatiko